labs/SettingsControls/src/SettingsCard/SettingsCard.xaml

```diff
@@ -237,88 +231,743 @@
-
-                            <StackPanel Grid.Column="1"
-                                        Margin="0,0,16,0"
-                                        HorizontalAlignment="Stretch"
-                                        VerticalAlignment="Center">
-
-                                <TextBlock x:Name="PART_HeaderPresenter"
-                                           VerticalAlignment="Center"
-                                           win:AutomationProperties.AccessibilityView="Raw"
-                                           win:HighContrastAdjustment="None"
-                                           Foreground="{TemplateBinding Foreground}"
-                                           Text="{TemplateBinding Header}" />
-
-                                <ContentPresenter x:Name="PART_DescriptionPresenter"
-                                                  win:AutomationProperties.AccessibilityView="Raw"
-                                                  Content="{TemplateBinding Description}"
-                                                  FontSize="12"
-                                                  Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                                  TextWrapping="WrapWholeWords"
-                                                  Visibility="Collapsed">
-                                    <ContentPresenter.Resources>
-                                        <Style BasedOn="{StaticResource CaptionTextBlockStyle}"
-                                               TargetType="TextBlock">
-                                            <Style.Setters>
-                                                <Setter Property="TextWrapping" Value="WrapWholeWords" />
-                                            </Style.Setters>
-                                        </Style>
-                                        <Style BasedOn="{StaticResource DefaultHyperlinkButtonStyle}"
-                                               TargetType="HyperlinkButton">
-                                            <Style.Setters>
-                                                <Setter Property="FontSize" Value="12" />
-                                                <Setter Property="Padding" Value="0,0,0,0" />
-                                            </Style.Setters>
-                                        </Style>
-                                    </ContentPresenter.Resources>
-                                </ContentPresenter>
-                            </StackPanel>
+                            <labs:HeaderedContentControl x:Name="PART_Descriptor"
+                                                         HorizontalAlignment="Left"
+                                                         VerticalAlignment="Center"
+                                                         Header="{TemplateBinding Header}"
+                                                         Icon="{TemplateBinding Icon}">
```

Comment:
@michael-hawker As discussed before.. overriding does not seem to work here. I am probably missing something obvious 😅.